skillful_network_client/src/app/registration-confirmation/registration-confirmation.component.html

```diff
@@ -0,0 +1,54 @@
+<div class="container">
+    <mat-card class="connectionForm">
```

Comment:
Could we reuse or create an abstract `mat-card` here? Or maybe it should be integrated into the `LoginComponent`.